libgadget/forcetree.h

```diff
@@ -76,13 +76,16 @@ typedef struct ForceTree {
     int *Father;
 } ForceTree;
 
+/*Initialize the internal parameters of the forcetree module*/
+void init_forcetree_params(const int FastParticleType, const double * GravitySofteningTable);
+
 int force_tree_allocated(const ForceTree * tt);
 
 /* This function propagates changed SPH smoothing lengths up the tree*/
 void force_update_hmax(int * activeset, int size, ForceTree * tt);
 
-/*This is the main constructor for the tree structure. Pass in something empty.*/
-void force_tree_rebuild(ForceTree * tree, DomainDecomp * ddecomp);
+/*This is the main constructor for the tree structure. Pass in something empty for tree.*/
```

Comment:
`tree shall be either zero-filled (due to force_tree_allocated), or a valid ForceTree.`